lib/info.dart

```diff
@@ -529,17 +539,17 @@ InfoKind kindFromString(String kind) {
 }
 
 /// A simple visitor for information produced by the dart2js compiler.
-abstract class InfoVisitor<T> {
-  T visitAll(AllInfo info);
-  T visitProgram(ProgramInfo info);
-  T visitLibrary(LibraryInfo info);
-  T visitClass(ClassInfo info);
-  T visitField(FieldInfo info);
-  T visitConstant(ConstantInfo info);
-  T visitFunction(FunctionInfo info);
-  T visitTypedef(TypedefInfo info);
-  T visitClosure(ClosureInfo info);
-  T visitOutput(OutputUnitInfo info);
+abstract class InfoVisitor<T, C> {
+  T visitAll(AllInfo info, C context);
```

Comment:
I thought about that, but since the `accept` methods of the `Info` classes must call the visit method and pass the context, they will always pass the context anyways